### ISSUE TITLE
assign different default keybinding for focusing the accessible buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#175162]
@jooyoungseo did a demo of his terminal flow for me in which he uses an editor to formulate terminal input, uses run the selected text in the active terminal, and then focuses the accessibility buffer to review the output. 

Since tab moves the focus and does special things in terminals (oh-my-zsh or ipython completions), we should consider a default keybinding that does not involve tab.  

This also means we can set this for non-screen reader users.

His keybindings are:

``` json
[
  {
    "key": "alt+`",
    "command": "workbench.action.terminal.focusAccessibleBuffer",
    "when": "accessibilityModeEnabled && terminalHasBeenCreated || accessibilityModeEnabled && terminalProcessSupported"
  },
  {
    "key": "shift+tab",
    "command": "-workbench.action.terminal.focusAccessibleBuffer",
    "when": "accessibilityModeEnabled && terminalFocus && terminalHasBeenCreated || accessibilityModeEnabled && terminalFocus && terminalProcessSupported"
  }
]
```

Currently, I have the following three issues:

1. alt+` does not work when my focus is in terminal input.

1. alt+` does not work when terminal is created in editor area.

1. Shift+Tab still does not cycle back through the ipython's auto-suggestion list even when tab to focus mode is off (AKA, tab insertion mode).

_Originally posted by @jooyoungseo in https://github.com/microsoft/vscode/issues/174800#issuecomment-1439215426_

**Comment by meganrogge** [comment]
I can reproduce the issue. When i change the keybinding to `alt+f` instead of `alt+backquote`, it works

**Comment by meganrogge** [comment]
we will just check tab focus mode instead of changing this
